man/systemd.exec.xml

```diff
@@ -381,6 +381,14 @@
         definitions. Each definition consists of a colon-separated tuple of source path and destination
         definitions, optionally followed by another colon and a list of mount options.</para>
 
+        <para>If the destination is empty, then a variable overlay is set up on the root. Nodes in the image
+        that are leaves in the root (empty directories or files) are bind-mounted individually.
+        Existing directories with differing content will be set up with a read-only overlayfs.
+        For example, if the root has <filename>/opt/</filename> and <filename>/var/foo</filename>, and the
+        image has <filename>/opt/bar</filename> and <filename>/var/baz</filename>, the end result will be
+        that <filename>/opt/</filename> from the image is bind-mounted on <filename>/opt/</filename> on the root,
+        and <filename>/var</filename> from the root and the image will both be part of an OverlayFS mount.</para>
```

Comment:
we typically suffix all dirs with / to indicate they are a dir. is /var/foo and /var/baz supposed to be a dir? (if not, it might be good to mention that somewhow

Reply:
they were files in the example - changed to specify files/directories explicitly